mission_summary/mission_summary.py

```diff
@@ -0,0 +1,209 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+
```

Comment:
The logos should go in a seperate directory. Traditionally this is called "static"  or "img" so we know that it is static images, not stuff that dynamically changes. See e.g. https://github.com/GliderToolsCommunity/GliderTools/tree/master/docs